src/components/CommonGridBasedForm/autoComplete/index.js

```diff
@@ -15,9 +22,12 @@ const AutoComplete = (props) => {
       id="combo-box-demo"
       onChange={onChange}
       options={values}
-      // eslint-disable-next-line react/jsx-props-no-spreading
       renderInput={(params) => <TextField {...params} label={placeholder} />}
+      // eslint-disable-next-line react/jsx-props-no-spreading
       sx={{ width: 300 }}
+      // eslint-disable-next-line react/jsx-props-no-spreading
+
+      value={labelValue ? labelValue[0].label : labelValue}
```

Comment:
Why are we using hardCoded index 0 is that fix we have to always get the value from label values at 0th index